docs/relying-parties/how-twos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+
+## Prerequisites
+
+This guide assumes you are already integrated with FxA/SubPlat; please see [Integration with Subscription Platform](/ecosystem-platform/relying-parties/tutorials/integration-with-subscription-platform) for more information.
+
+Ideally, you will also want to grant 1-2 SubPlat engineers working on the integration developer access to your android app in [Google Play Console](https://play.google.com/console) ([example bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1796624)). This allows us to access your android app’s [Google Play Developer API](https://developers.google.com/android-publisher) keys, to look up needed app-specific information and to configure an endpoint to receive [Real-time developer notifications](https://developer.android.com/google/play/billing/rtdn-reference).
+
+## Configuration
+
+These steps can be done by a product manager or developer and in parallel with steps in [SubPlat API Calls](#subplat-api-calls). More information regarding specific Stripe metadata keys mentioned below can be found in the [Subscription Platform integration tutorial](/ecosystem-platform/tutorials/subscription-platform#stripe-productplans).
+
+### Create an IAP configuration document
+
+This document represents a list of Play Store plans (identified uniquely by their Play Store [`token`](https://developer.android.com/google/play/billing#tokens) for your product.
+
+1. Go into Google Play Console and find details about your android app by [`token and orderId`](https://developer.android.com/google/play/billing#tokens).
+2. Create a configuration document similar in form to https://api.accounts.firefox.com/v1/oauth/subscriptions/iap/plans/guardian-vpn (omitting iOS plans) using the details retrieved in Step 1. This can be in a Google document for now.
+   - Note: The only information SubPlat uses from the VPN document currently is the `productId` and `platform`. The VPN configuration document is structured in this way for historical reasons. We may change how this information is structured for future integrations.
+3. File a ticket in the [FXA Jira project](https://mozilla-hub.atlassian.net/browse/FXA) to create a new document in SubPlat's "IAP config" Firestore collection, and provide the document from Step 2 along with an `appName`.
+   - `appName` is a human-readable name that SubPlat uses to uniquely identify your app in our system (e.g. `"guardian-vpn"` is the `appName` for the Mozilla VPN in the link above).
+     When the ticket is complete and deployed, you should be able to view your configuration at https://api.accounts.firefox.com/v1/oauth/subscriptions/iap/plans/${appName}.
+
+### Add Google Play Developer API keys in SubPlat
+
+A SubPlat engineer with developer access to Google Play Console (see [Prerequisites](#prerequisites)) can configure this for you when ready.
```

Comment:
As mentioned above, I want to make sure the Google Play Console is where developers should go to get these keys. Hopefully Ben can confirm. We'll want to find and replace if it's some other place.

Reply:
I updated this section and added this link instead: https://support.google.com/googleapi/answer/6158862 since the doc you linked doesn't quite do the actual steps involved justice.